pom.xml

```diff
@@ -804,8 +803,7 @@
           <plugin>
             <groupId>org.apache.maven.plugins</groupId>
             <artifactId>maven-surefire-plugin</artifactId>
-            <!-- Must use older version of surefire plugin for native-image testing. -->
-            <version>2.22.2</version>
```

Comment:
Thanks for the upgrade @JoeWang1127! Would it be possible to do a quick check with a few handwritten libraries?  (pubsub, spanner and bigquery, for example) I remember this causing some issues a while back when I tried it out so. what to double check if this is still a problem. We could create a draft PR in those libs, copy-paste this profile, rename it to something like `native-2` and modify the `.kokoro/build.sh` to use `-Pnative-2`)

Reply:
Sure, when you think the profile is OK, I'll verify it in handwritten libraries.